documentation/slate/source/includes/realtime.md

```diff
@@ -188,7 +248,8 @@ See the [disruption](#disruption) objects section for its full content and descr
 
 ### Journeys
 
-If the station deleted is the destination of the journey, Navitia will compute an other itinerary to the requested station. If not, the disruption will be present at the root level of the response for information, but it won't be affecting the journey.
+If the stop deleted is the origin/destination of a section of the journey, Navitia will compute a different itinerary to the requested station.  
+If not, the disruption will be present in the `display_informations.links` part of the section for information, but it won't be affecting the journey.
```

Comment:
Is it not useful to have an extract from the response to show this? Maybe not because it's already in the previous example of /journeys?!

Also, for consistency, you wrote `display_informations.links` and I wrote A link to this disruption can be found in the section “display_informations”. Let's decide which to way to write it and put it everywhere.